plugins/Login/lang/en.json

```
@@ -69,6 +79,10 @@
         "AcceptPrivacyPolicy": "You need to accept the privacy policy.",
         "AcceptTermsAndCondition": "You need to accept the terms &amp; conditions.",
         "AcceptPrivacyPolicyAndTermsAndCondition": "You need to accept the privacy policy and the terms &amp; conditions.",
-        "CreatePasswordDescription": "Make sure you remember this password and keep it in a safe place."
+        "CreatePasswordDescription": "Make sure you remember this password and keep it in a safe place.",
+        "CancelPasswordResetRequestRemoved": "Your request has been removed",
```

Should that be 
```suggestion
        "CancelPasswordResetRequestRemoved": "The request has been removed",
```

Logically the user had previously clicked on a link "That wasn't me", so we know that it was not their request 🙈